keps/sig-network/3866-nftables-proxy/README.md

```diff
@@ -1720,15 +1679,15 @@ question.
 
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
-TBD.
+`sync_proxy_rules_nftables_sync_failures_total` indicates the number
```

Comment:
please fill in the SLO question above.

[I would be fine with just saying that for now we're sticking to the network programming latency (https://github.com/kubernetes/community/blob/master/sig-scalability/slos/network_programming_latency.md ) and we will reevaluate that before GA.